src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/mapper/induction/QualificationsAndTrainingResourceMapper.kt

```diff
@@ -65,7 +65,7 @@ abstract class QualificationsAndTrainingResourceMapper {
     }
     return EducationAndQualificationResponse(
       id = training.reference,
-      educationLevel = toHighestEducationLevelApi(qualifications?.educationLevel),
+      educationLevel = toHighestEducationLevelApi(qualifications?.educationLevel) ?: HighestEducationLevelApi.NOT_SURE,
```

Comment:
In addition to defaulting this in the request (for future create/update requests), I'm now defaulting this in the response for existing entries. I didn't rush this through last night, but I'm confident the UI handles it (for both journeys) so I'm including it now.